keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -503,6 +555,8 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+N/A.
```

Comment:
this is always in  use when creating PVCs from VolumeSnapshotContents correct?

Reply:
Changed